services-toolkit/reference/api/rbac.hbs.md

```diff
@@ -0,0 +1,31 @@
+# RBAC
```

Comment:
**suggestion** We should include an example of the RBAC claim verb here

Reply:
Thanks - have added this here (also linked it to the more-detailed how-to guide). Also added RBAC labels for aggregation to the providers.